### ISSUE TITLE
[No connection!!]

### ISSUE THREAD

**Issue by StrikeSNC** [chris1111/D-LinkUtility-Package#7]
Sorry for disturbing in holidays. I've been trying to make this work for a while but still no luck.
Although I was successful to install the driver apparently, it doesn't seems to be able to connect to my network. I'm sure that my network details are correct.
Where can I edit the .plist file after I've installed the utility? Or, where can I check the status of connection of which part is wrong?
I am currently using an MT7601 chip which prehaps good in Catalina and installed the utility under the /L/E/Extensions way. Thank you for you patience.

**Comment by chris1111** [comment]
Check infos from **ActiveProfileSetting.plist** in /HOME/Library/Preferences
Verifying if all the infos is correct then restart after editing the plist file

**Comment by StrikeSNC** [comment]
Thanks for the quick respond. The information seems to be correct though, however it is still unable to connect.
Is there any way I can check that which part has gone wrong in the connection progress? Thanks.

**Comment by chris1111** [comment]
Do you have a snapshot disk on this macOS?

**Comment by StrikeSNC** [comment]
I am not sure. It's a fresh install of macOS 12.1.

**Comment by StrikeSNC** [comment]
Update: To verify if it's my fault on editing .plist or not, I've specifically installed an older macOS version (Mavericks) which works with the previous package in a VM and mounting my Wi-Fi USB here, and it does connect to my Wi-Fi. However, when I copy the Profile to my Monterey Hackintosh, it still shows that there is no connection. This is actually frustrating. I don't know if this matters or not, but the .plist name in the VM is called "Profile.plist" instead of "ActiveProfileSetting.plist".

**Comment by chris1111** [comment]
we have two plist
ActiveProfileSetting.plist, and AdvancedProfileSetting.plist

**Comment by StrikeSNC** [comment]
I've replaced contents in both plist from the VM that I can connect to my Wi-Fi, however it still has no luck.
I think when I updated Catalina to Big Sur before you make the installer which replaces content on snapshot disk, I can still connect to the Wi-Fi but just not using the perfpane. Quite interesting.

**Comment by vkocev** [comment]
I can confirm that it dosen't work on Monterey 12.1 (21C52).  I have 2 usb dongles which work on High Sierra.  I tried everything, SIP disable/enable.  It loads the kext, I can see the adapters in System Prefrences>Network.  But the app says No Connection. I am sure I am editing the plist just fine. I edited the ActiveProfile.plist and Advnaced manually but still no luck.  Please explain which MAC address should I put in BSSD?  Is this the Acess Point MAC address (of the WIFI router) or the Ethernet en0 or the wifi dongle MAC address.  My ethernet is en0.  The wifi dongles are shown as en2 and en3 in System Information>Network.  Please also explain what are the preferences for WPA2 and also for OPEN netwrok without password.  In terms of Encryption Type, Authenticaion type, Network Mode (mode 9), etc (PSK+TKIP, etc). I see mode 13 in High Siera mediatek app. It is easier to connect without password and test from there. Maybe the error is somewhere in these parameters.  Please note that the driver I use in High Sierra is version 5.0.1.2.2 and it is larger kext file, some 4MB and yours (5.0.1.35) kext is 1.5 MB.  Can I use the driver from High Sierra?  Thanks

**Comment by vkocev** [comment]
@StrikeSNC please share your working profile from Maverics.  I want to try it please.  Thanks

**Comment by StrikeSNC** [comment]
I wish I could've help, but I am not sure why when it works on my Mavericks VM but not the Monterey host installation.

**Comment by vkocev** [comment]
@StrikeSNC Can you please drag and drop it in the reply, the Maverics "Profile.plist" or just copy/paste it.  Delete your credentials before.  I would like to compare it please with the parametars from ActiveProfileSetting.plist.  Thanks

**Comment by davgusm** [comment]
Did you manage to solve it? I have this same problem of my USB Wifi not connecting and [No Connection]

**Comment by chris1111** [comment]
I do not understand the problem here because another test on my part with the AWUSO36ACM adapter this time and works wonderfully on Monterey 12.3 (21E230)
![Screen Shot 2022-03-25 at 10 54 16 AM](https://user-images.githubusercontent.com/6248794/160148460-b1ea07ee-c865-482c-9ae0-4dd6d31033c5.png)


Are you sure you have entered the MAC Address by removing all the two dots
Not Good ➤ **ec:b1:d7:2d:86:e0**
Good ➤ **ecb1d72d86e0**

**Comment by davgusm** [comment]
Which MAC Address I use, in your video it is not clear where you get it. is it from Ethernet? or the WiFi dongle?

**Comment by StrikeSNC** [comment]
Maybe I might did something wrong. Let me try again later. Although I am pretty sure I have entered the correct MAC address without the dots, from the "802.11 n WLAN" connection.

However, are you planning of porting the old Big Sur project where it breaks the seal to Monterey? I've tried it on 12.3 and seems like the kext is working while the panel isn't.

**Comment by StrikeSNC** [comment]
<img width="794" alt="Screenshot 2022-03-26 at 00 01 58" src="https://user-images.githubusercontent.com/40461728/160158045-13a6b7e9-1be2-4ffc-be44-49ec5e798042.png">
<img width="1424" alt="Screenshot 2022-03-26 at 00 06 55" src="https://user-images.githubusercontent.com/40461728/160158444-96b9857d-9364-4b8a-a539-caa7df07631d.png">


Tried again on 12.3 (21E230), doesn't work. Here's my plist. I am sure that my network name and passwords are both correct.

**Comment by chris1111** [comment]
select the Ethernet Mac adress

**Comment by StrikeSNC** [comment]
This is the only "Ethernet" connection I have.

**Comment by chris1111** [comment]
> This is the only "Ethernet" connection I have.


Why?

**Comment by davgusm** [comment]
where do you edit this ActiveProfileSettings.plist file?

**Comment by chris1111** [comment]
> where do you edit this ActiveProfileSettings.plist file?

use the App thats it

**Comment by StrikeSNC** [comment]
> > This is the only "Ethernet" connection I have.
> 
> Why?

Laptop with no ethernet port. There is a so-called ethernet adapter reported, but it is for the external dock use only.
<img width="666" alt="image" src="https://user-images.githubusercontent.com/40461728/160161044-507e5196-9f59-4c80-b11e-84fb72f66164.png">

**Comment by StrikeSNC** [comment]
> where do you edit this ActiveProfileSettings.plist file?

/Users/<User>/Library/Perferences, open with TextEdit.

**Comment by chris1111** [comment]
why editing ActiveProfileSettings.plist if you use the app 
This is probably the issue here, if you use this program without the readme dont open issue
This issue will be deleted !
